### PR TITLE
Table: Allow translated strings for pagination and sorting

### LINKED ISSUE #2108: [Modus-Table] Allow translated strings for pagination and sorting.
### Prerequisites

- [X] I have [searched](https://github.com/trimble-oss/modus-web-components/issues?q=is%3Aissue) for duplicate or closed feature requests
- [X] I have read the [Modus style guidelines](https://modus.trimble.com/components/web/introduction/)

### Proposal

The modus-table component contains some hardcoded English strings. As shown in the below screenshot:
image

As a developer I would like to be able to provide custom values for these strings, so that I can support multiple languages.

### Motivation and context

We are planning to release a UI component, that uses the modus-table, which will be used in a multi-language product.

### What is the issue regarding ?

@trimble-oss/modus-web-components

### Priority

High

### What product/project are you using Modus Components for ?

Nova / Construction calculation platform UI

### What is your team/division name ?

ACDC

### Are you willing to contribute ?

Maybe

### Are you using Modus Web Components in production ?

_No response_

### PR DESCRIPTION
**Pull request by prashanthr6383**
## Description

References 

Fixes #2108 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Comment by jewel-shajan** [comment]
Testing has been completed. Please confirm whether we need to translate the 'Please select' also.
image

**Comment by prashanthr6383** [comment]
> Testing has been completed. Please confirm whether we need to translate the 'Please select' also. image

@jewel-shajan added translations for select component

File: stencil-workspace/src/lang/translations.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
+interface Translation {
+  [key: string]: string;
+}
+
+const translations: Record<string, Translation> = {
+  en: {
+    showingResult: 'Showing result',
+    of: 'of',
+    pageView: 'Page View',
+    pleseSelect: 'Please select',
```
Comment (coliff) [inline stencil-workspace/src/lang/translations.ts:10]: Looks like a typo - can you change `pleseSelect` to `pleaseSelect` ?

**Comment by jewel-shajan** [comment]
Testing Completed and ready for PR Review.

**Review by saravananrmsmy — APPROVED** [review]

**Pull request merged**